README.md

```diff
@@ -73,6 +73,31 @@ cd gitleaks
 make build
 ```
 
+### Load Configuration
+
+The order of precedence is:
+
+1. `--config/-c` option:
+  ```bash
+  gitleaks git --config /home/dev/customgitleaks.toml .
+  ```
+1. Environment variable `GITLEAKS_CONFIG` with the file path:
+  ```bash
+  export GITLEAKS_CONFIG="/home/dev/customgitleaks.toml"
+  gitleaks git .
+  ```
+1. Environment variable `GITLEAKS_CONFIG_TOML` with the file content:
+  ```bash
+  export GITLEAKS_CONFIG_TOML=`cat customgitleaks.toml`
+  gitleaks git .
+  ```
+1. A `.gitleaks.toml` file within the target path:
+  ```bash
+  gitleaks git .
+  ```
```

Comment:
After playing with this locally, it seems the code blocks need to be indented another level:
````suggestion
1. `--config/-c` option:
      ```bash
      gitleaks git --config /home/dev/customgitleaks.toml .
      ```
2. Environment variable `GITLEAKS_CONFIG` with the file path:
      ```bash
      export GITLEAKS_CONFIG="/home/dev/customgitleaks.toml"
      gitleaks git .
      ```
3. Environment variable `GITLEAKS_CONFIG_TOML` with the file content:
      ```bash
      export GITLEAKS_CONFIG_TOML=`cat customgitleaks.toml`
      gitleaks git .
      ```
4. A `.gitleaks.toml` file within the target path:
      ```bash
      gitleaks git .
      ```
````